### DISCUSSION TITLE
Using Shallow-render unit tests with Transloco

### DISCUSSION THREAD

**Discussion by joelynch-1** [ngneat/transloco#660] [Q&A]
I have an angular project where I use shallow for some of my unit tests. The setup is something like:

`shallow = new Shallow(myComponent, myModule)`. 

After converting my project to use transloco, these unit tests fail because I have not provided them the TranslocoTetsingModule. However, I can not just replace myModule with the TranslocoTetsingModule. Does anyone have any ideas/solutions to this?

**Comment by shaharkazaz** [comment]
Can you please add some more context? maybe a reproduction link? a link to the shallow package you are using?
